packages/core/src/node/messaging/ipc-channel.ts

```diff
@@ -48,13 +49,12 @@ export class IPCChannel implements Channel {
 
     protected ipcErrorListener: (error: Error) => void = error => this.onErrorEmitter.fire(error);
 
-    constructor(pipe: Duplex, childProcess?: cp.ChildProcess) {
```

Comment:
Do we still want to pass the duplex to listen to as argument?

Reply:
I don't think that is necessary.